pagerduty/resource_pagerduty_event_orchestration_path_unrouted_test.go

```diff
@@ -0,0 +1,477 @@
+package pagerduty
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("pagerduty_event_orchestration_unrouted", &resource.Sweeper{
+		Name: "unrouted",
+		Dependencies: []string{
+			"pagerduty_schedule",
+			"pagerduty_team",
+			"pagerduty_user",
+			"pagerduty_escalation_policy",
+			"pagerduty_service",
+			"pagerduty_event_orchestration",
+		},
+	})
+}
+
+func TestAccPagerDutyEventOrchestrationPathUnrouted_Basic(t *testing.T) {
+	team := fmt.Sprintf("tf-name-%s", acctest.RandString(5))
+	escalation_policy := fmt.Sprintf("tf-%s", acctest.RandString(5))
```

Comment:
Can we rename `escalation_policy` to `escalationPolicy` to match Go naming conventions?

Reply:
Oh, my bad🙂. Forgot about this one. Changing right now

Reply:
Done